my-app/src/App/components/Line/Line.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import styles from "./Line.module.scss";
+
+const Line = () => {
+  return <div className={styles.line}></div>;
```

Comment:
если у jsx-тега нет `children` можно сделать его самозакрывающимся
```suggestion
  return <div className={styles.line} />;
```